docs/syntax.rst

```diff
@@ -2,9 +2,14 @@
 The Guide To The FTL Syntax
 ===========================
 
-FTL is an open source, localization-specific scripting language used to process 
-gender, plurals, conjugations, and most of the other quirky elements of natural 
-language.
+FTL is a localization language used for describing translation resources.
```

Comment:
OK, I'm not going to block on this but keep in mind that in our domain the word `language` is ambiguous and there's a good chance people will think about the natural language here.  How about `file format`?